js/jquery.mapael.js

```diff
@@ -202,7 +202,7 @@
 
                             if (Math.abs(pinchDist - previousPinchDist) > 15) {
                                 offset = $map.offset();
-                                initFactor = (options.map.width) ? (Mapael.maps[options.map.name].width / options.map.width) : ($.fn[pluginName].maps[options.map.name].width / $map.width());
+                                initFactor = (options.map.width) ? (mapConf.width / options.map.width) : ($.fn[pluginName].maps[options.map.name].width / $map.width());
```

Comment:
Great fix, I think that `$.fn[pluginName].maps[options.map.name].width` could also be replaced by `mapConf.width` here !